app/experimenter/projects/tests/test_views.py

```diff
@@ -11,3 +14,58 @@ def test_list_view_returns_projects(self):
         }
         response = self.client.get(reverse('projects-list'), **headers)
         self.assertEqual(response.status_code, 200)
+
+
+class TestProjectCreateView(TestCase):
+
+    def test_project_create_view_creates_project_and_redirects(self):
+        headers = {
+            settings.OPENIDC_EMAIL_HEADER: 'user@example.com',
+        }
+
+        project_data = {
+            'name': 'My Project',
+        }
+
+        response = self.client.post(
+            reverse('projects-create'), project_data, **headers)
+
+        project = Project.objects.get()
+
+        self.assertEqual(project.name, project_data['name'])
+
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(
+            response.url,
+            reverse('projects-detail', kwargs={'slug': project.slug}),
+        )
```

Comment:
You can simplify this with [`self.assertRedirects`](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.assertRedirects)